src/Basisregisters.IntegrationDb.NationalRegistry/FilesOutput.cs

```diff
@@ -170,5 +165,123 @@ byte[] TransformRecord(AddressWithRegisteredCount addressWithRegisteredCount)
                 () => addressesWithRegisteredCount.Count,
                 TransformRecord);
         }
+
+        public static void WriteDbfFile(IList<UnmatchedFlatFileRecord> unmatchedRecords, string directory)
```

Comment:
Can we move the overloads together?

Reply:
I didn't on purpose so that they stay together per entity (I've seen the sonarcloud warning)